test/Azure.Functions.Cli.Tests/Azure.Functions.Cli.Tests.csproj

```diff
@@ -22,7 +21,7 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="Xunit.SkippableFact" Version="1.3.6" />
-    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.95-preview" />
+    <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.102-preview" />
```

Comment:
Just need to remove: `<PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.95-preview" />`
Don't need to add `<PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.102-preview" />`

Reply:
removed